### ISSUE TITLE
"CTRL+" and "CTRL-" should work for all UI texts

### ISSUE THREAD

**Issue by lfcnassif** [sepinf-inc/IPED#679]
This is not working for the options menu and dialog windows (bookmarks, column and filter managers, file/folder selection windows, tooltips, gallery col count and other dialogs).

**Comment by lfcnassif** [comment]
I'm not sure if this is worth to fix, given it will apply just for texts, not for all components, or if this feature could be removed after #48 is implemented.

**Comment by tc-wleite** [comment]
@lfcnassif, the current implementation (CRTL+/-) is very simple, and I am also not sure if it is worth fixing.
I need to check in practice, but I believe #48 (at least a simple version of it) can be implemented in a similar way dark mode feature #569 was.
As you are probably dealing with other stuff, I can take a closer look and see if there is a straightforward solution to #48.

**Comment by lfcnassif** [comment]
Thanks, @tc-wleite. But don't worry, this is low priority.

**Comment by tc-wleite** [comment]
As discussed in #48, this feature will be removed.  
Instead, Java uiScale wil be used, when full support to Java newer versions (>= 9) is added.
